plugin/PDFDocumentPage.qml

```diff
@@ -27,6 +27,10 @@ import "PDFStorage.js" as PDFStorage
 DocumentPage {
     id: base
 
+    property alias document: pdfDocument
+    property alias placeHolder: placeHolderLoader
+    property alias toolbar: toolbar
+
```

Comment:
I try to tune the tests so added accessors may have a sense from a public API point of view, like exposing the toolbar item so any external user of the PDFDocumentPage item may control better the toolbar behaviour for their own needs.

Well, for the moment, I've simply added a comment on every added accessors…